voucher/utils.py

```diff
@@ -95,7 +103,10 @@ def get_eligible_coupon_choices(voucher):
             json.dumps(
                 (course_matches[i].product.first().id, valid_coupons[i].coupon.id)
             ),
-            course_matches[i].title,
+            "{title} - {start_date}".format(
```

Comment:
I'd like to make the date more readable. 

Here's an easy change:

```suggestion
            "{title} - starting {start_date}".format(
```

Perhaps a hard one would be to format 2019-08-20 as Aug 8, 2019.

I'd like to get this change out as soon as possible, so let's go with whatever is fastest.

Reply:
@pdpinch both options are easy, confirm the following format.

`Additive Manufacturing for Innovative Design and Production - starts Aug 20, 2019`